project/Settings.scala

```diff
@@ -72,6 +72,8 @@ object Settings {
     "-Ywarn-unused:implicits",
     "-Ywarn-unused:privates",
     "-Ywarn-unused:locals",
+    "-Ycache-macro-class-loader:last-modified",
```

Comment:
Were these the "experimental, try these out maybe and let us know how it goes" options mentioned in NE Scala?

Reply:
I didn't hear or see "experimental" in the NEscala talk or github release, respectively.  I'm waiting to see if the build explodes but otherwise this should be a "maybe it will help but it won't hurt" scenario.